src/random_mc.jl

```diff
@@ -81,70 +81,118 @@ end
 # random_stochastic_matrix
 
 """
-Return a randomly sampled n x n stochastic matrix.
+Return a randomly sampled n x n stochastic matrix with k nonzero entries for
+each row.
 
 ##### Arguments
 
 - `n::Integer` : Number of states.
-- `k::Integer` : Number of nonzero entries in each row of the matrix.
+- `;k::Union{Integer, Void}(nothing)` : Number of nonzero entries in each
```

Comment:
This doesn't appear to be a keyword argument, which is what we reserve the leading `;` syntax for.

I suggest either removing the semicolon here or making `k` a keyword argument